client/starwhale/api/_impl/argument.py

```diff
@@ -94,14 +65,25 @@ def asdict(self) -> t.Dict[str, t.Any]:
         for func, dtypes in self._func_related_dataclasses.items():
             for dtype in dtypes:
                 for option in self._options[dtype]:
-                    field = OptionField(**option.to_info_dict())
+                    field = OptionFieldClient(**option.to_info_dict())
                     if PYDANTIC_V2:
                         info = field.model_dump(mode="json")
                     else:
                         info = field.dict()
                     r[func][dtype][option.name] = info
         return r
 
+    def asobj(self) -> t.Dict[str, t.Dict[str, t.Dict[str, OptionField]]]:
```

Comment:
define a type alias for `t.Dict[str, t.Dict[str, OptionField]]`?  StepSpec.arguments field also needs this type.

Reply:
done，but StepSpec.arguments was generated by pydantic, so just used in argument.py